src/modules/HomePage/ShopByCategory/ShopByCategory.module.scss

```diff
@@ -0,0 +1,133 @@
+@import '../../../styles/blocks/grid';
+@import '../../../styles/utils/variables/colors';
+@import '../../../styles/utils/mixins/mixin-media';
+
+.shopByCategory {
+  padding-block: 57px;
+  padding-inline: 16px;
+  max-width: 1200px;
+  margin: 0 auto;
+  @include onTablet {
+    margin-block: 64px;
+    padding-inline: 24px;
+  }
+
+  &__title {
```

Comment:
in module scss nesting is mostly not used, to prevent mess, only in cases with modificators or pseudo classes/ elements